include/sys/fs/zfs.h

```diff
@@ -240,6 +240,7 @@ typedef enum {
 	ZPOOL_PROP_MAXDNODESIZE,
 	ZPOOL_PROP_MULTIHOST,
 	ZPOOL_PROP_CHECKPOINT,
+	ZPOOL_PROP_LOAD_GUID,
```

Comment:
If we stick with the proposed naming scheme we should add a `ZFS_PROP_ID` dataset property.  Otherwise users will need to resort to zdb to map dataset names to their matching kstat entry.

Reply:
What about this?
```
$ zfs get -Hp -o value objsetid rpool/swap
203
```

Reply:
Yes, that's exactly what we need!  Since it's a hidden property I didn't even realize that existed, we may want to unhide it and add it to the documentation.

Reply:
Review update made this a normal property